bokehjs/src/coffee/core/util/serialization.coffee

```diff
@@ -15,12 +15,12 @@ for k, v of ARRAY_TYPES
     DTYPES[v.name] = k
 
 _arrayBufferToBase64 = (buffer) ->
-  binary = ''
+  binary = []
   bytes = new Uint8Array( buffer )
   len = bytes.byteLength
   for i in [0...len]
```

Comment:
Just to note, unless you actually need the index, you can safely write more readable yet efficient code for this in CoffeeScript, i.e.:
```
for b in bytes
 binary.push(String.fromCharCode(b))
```
or even:
```
binary = (String.fromCharCode(b) for b in bytes)
```
One of few good features of CoffeeScript are its `for` loops which get translated into efficient `for(var i, ...)` loops.